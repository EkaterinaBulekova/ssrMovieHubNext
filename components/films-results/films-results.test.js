import React from 'react';
import { shallow } from 'enzyme';
import FilmsResults from './films-results';
import toJson from 'enzyme-to-json';

describe('<FilmsResults />', () => {
  it('should render with right count and buttons with onClick method', () => {
    const countClass = "films-count";
    const testCount = 5;
    const testOnClick = jest.fn();
    const component = shallow(<FilmsResults/>);
    expect(toJson(component)).toMatchSnapshot();

    component.unmount();
  });
});