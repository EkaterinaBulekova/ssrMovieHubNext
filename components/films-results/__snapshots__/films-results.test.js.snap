// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilmsResults /> should render with right count and buttons with onClick method 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
